include/fmt/format.h

```diff
@@ -62,6 +62,10 @@
 #if FMT_GCC_VERSION >= 406 || FMT_CLANG_VERSION
 #  pragma GCC diagnostic push
 
+// Disable warning about not handling all enums in switch statement even with
+// a default case
+#  pragma GCC diagnostic ignored "-Wswitch-enum"
```

Comment:
Could you post specific warnings this is causing? Might be better to fix those instead of suppressing.

Reply:
![screenshot_2019-02-23_13-29-00](https://user-images.githubusercontent.com/1961592/53290235-05388b00-376f-11e9-8eca-975982a2f5a6.png)

I thought having a default case is cleaner and just disabling this warning is preferable.